packages/itwinui-css/src/anchor/mixins.scss

```diff
@@ -2,30 +2,6 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use '../mixins';
 
-@mixin iui-anchor {
-  &:where(button) {
-    font: inherit;
-    border: none;
-    background: transparent;
-    padding: 0;
-  }
-
-  @include iui-anchor-status('accent');
-  border-radius: var(--iui-border-radius-1);
-  cursor: pointer;
-
-  @include iui-anchor-underline('on-hover');
-
-  @media (prefers-contrast: more) {
-    @include iui-anchor-underline('on-initial');
-  }
-
-  &[data-iui-underline='true'],
-  :is([data-iui-contrast='high'] &) {
-    @include iui-anchor-underline('on-initial');
-  }
-}
-
 @mixin iui-anchor-status($status: accent) {
```

Comment:
These other mixins should also be removed. Everything should be controllable using plain CSS/HTML.

Reply:
Done in latest [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/63c95e30856b494b4444148e0b90542656c6ddb9)

Reply:
For `iui-alert-link` and `iui-toast-anchor`, you can use the `underline` prop in React and the `data-iui-underline` attribute in HTML, instead of repeating the underline CSS.

Also you can add new data attribute selector to `.iui-anchor` to reduce the repetition around status colors. The HTML would then become something like `<a class="iui-anchor iui-alert-link" data-iui-status="positive">`.

Reply:
Okay, will do.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/882a3b0dc56a27a8668d6b79dff78bdedeb4b9fd) Changed the `<ButtonBase>` to `<Anchor>` in the `Alert` component to use the `underline` prop, but couldn't do so with `Toast`, since the `link` object that gets spread across the button there takes in `HTMLButtonProps`. Instead, passed in `data-iui-underline` to the button. Could also add an `underline` button to prop, but I decided against it since we are trying to limit changes to react users.